pkg/probe/system_interface.go

```diff
@@ -64,6 +64,7 @@ func probeSystemInterface(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.
 		RxBytes   float64 `json:"rx_bytes"`
 		TxErrors  float64 `json:"tx_errors"`
 		RxErrors  float64 `json:"rx_errors"`
+		VlanID  int `json:"vlanid"`
```

Comment:
'json:"vlanid"' is an int

Error: json: cannot unmarshal number into Go struct field ifResult.Results.vlanid of type string